src/helpers.ts

```diff
@@ -0,0 +1,11 @@
+export const addPathToUrl = (customNetworkUrl: string, linkType: string, suffixType?: string,) => {
+    const { username, password, protocol, host, pathname, search, hash } = new URL(customNetworkUrl)
+
+    const newPath = pathname.endsWith('/') ? `${pathname}${linkType}/${suffixType}` : `${pathname}/${linkType}/${suffixType}` 
+    
+    const auth = username ? `${username}:${password}` : ''
+    
+    let parsedUrl = new URL(`${protocol}//${auth}${host}${newPath}${search}${hash}`)
+
+    return parsedUrl.toString();
+}
```

Comment:
Nit: Indentation, newline, and use of `let` for a `const` here as well. I've fixed these, and removed trailing whitespace on the empty lines:
```suggestion
  const { username, password, protocol, host, pathname, search, hash } = new URL(customNetworkUrl)

  const newPath = pathname.endsWith('/') ? `${pathname}${linkType}/${suffixType}` : `${pathname}/${linkType}/${suffixType}` 

  const auth = username ? `${username}:${password}` : ''

  const parsedUrl = new URL(`${protocol}//${auth}${host}${newPath}${search}${hash}`)

  return parsedUrl.toString()
}

```